staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -831,6 +834,122 @@ func TestTransformationFailure(t *testing.T) {
 	}
 }
 
+func TestKvsSplit(t *testing.T) {
+	testCases := []struct {
+		desc           string
+		kvs            []*mvccpb.KeyValue
+		numberOfChunks int
+		want           [][]*mvccpb.KeyValue
+	}{
+		{
+			desc:           "empty input results in nil - can't split an empty slice",
+			kvs:            []*mvccpb.KeyValue{},
+			numberOfChunks: 4,
+			want:           nil,
+		},
+		{
+			desc:           "input slice on len one",
+			kvs:            []*mvccpb.KeyValue{{Key: []byte("key_one")}},
+			numberOfChunks: 4,
+			want:           [][]*mvccpb.KeyValue{{&mvccpb.KeyValue{Key: []byte("key_one")}}},
```

Comment:
can you skip the type name for inner struct literal here?

Reply:
Done.